config/routes.rb

```diff
@@ -36,6 +36,9 @@
   end
 
   # ***** COURSES AND LESSONS ROUTES *****
+  # deprecated /courses/curriculum route redirect
+  get '/courses/curriculum' => redirect('/courses')
```

Comment:
can we put this redirect along with the other redirects at the bottom of the file please @narekvardanjan 😄